src/hw2/HW2.java

```diff
@@ -0,0 +1,31 @@
+package hw2;
+
+import java.util.Scanner;
+
+public class HW2 {
+
+    public static void main(String[] args) {
+
+        int a = 123;
+        System.out.println("Сума цифр числа " + a + " дорівнює " + ((a % 10) + ((a / 10) % 10) + ((a / 100) % 10)));
+
+
+        String Str = new String("Hello World");
```

Comment:
According to the home task there should be 2 words as 2 String parameters in a method